packages/validation/src/MonokleValidator.test.ts

```diff
@@ -0,0 +1,139 @@
+import { it, expect } from "vitest";
+import { ResourceParser } from "./common/resourceParser.js";
+import { Resource } from "./common/types.js";
+import { MonokleValidator } from "./MonokleValidator.js";
+import { LabelsValidator } from "./validators/labels/validator.js";
+
+it("should be easy to configure", async () => {
+  const validator = new MonokleValidator([LabelsValidator]);
```

Comment:
I'll have to revise this as it already doesn't hold up once I add OpenPolicyAgentValidator into the mix.. Since it's an internal package we can leave a nice DX interface as a low priority.